lang/src/org/partiql/lang/planner/PartiQLPlanner.kt

```diff
@@ -0,0 +1,31 @@
+package org.partiql.lang.planner
+
+import org.partiql.lang.domains.PartiqlAst
+import org.partiql.lang.domains.PartiqlPhysical
+import org.partiql.lang.errors.Problem
+
+interface PartiQLPlanner {
+
+    fun plan(statement: PartiqlAst.Statement): Result
+
+    companion object {
+        const val PLAN_VERSION = "0.0"
```

Comment:
Considering that logical plan gets versioned at the moment, curious what do we see for the utility of the Plan version, is that to version the `Result`?

Reply:
This value is used when constructing the logical plan, but I'm unaware of any IR versioning story

Reply:
What I meant was: we already version the plan using [PLAN_VERSION_NUMBER](https://github.com/partiql/partiql-lang-kotlin/blob/734bb26e930325a663686cc71abbb459a2f7886f/lang/src/org/partiql/lang/planner/transforms/Util.kt#L30) [here](https://github.com/partiql/partiql-lang-kotlin/blob/734bb26e930325a663686cc71abbb459a2f7886f/lang/src/org/partiql/lang/planner/transforms/AstToLogicalVisitorTransform.kt#L26). Since `PLAN_VERSION_NUMBER` is still included in this PR I was curious why are we having both?

Reply:
This PR intentionally does not remove any old code. After its merge, a new PR will be created to remove the old stuff.